include/nbl/video/IPhysicalDevice.h

```diff
@@ -148,49 +328,80 @@ class IPhysicalDevice : public core::Interface, public core::Unmovable
             // Whether `shaderSubgroupQuad` flag refer to all stages where subgroup ops are reported to be supported.
             // See SLimit::subgroupOpsShaderStages.
             bool shaderSubgroupQuadAllStages = false;
-            bool drawIndirectCount = false;
-            bool multiDrawIndirect = false;
-            bool samplerAnisotropy = false;
-            bool geometryShader    = false;
 
-            // RayQuery
+            //--> VkPhysicalDeviceRayQueryFeaturesKHR
             bool rayQuery = false;
 
-            // AccelerationStructure
+            //--> VkPhysicalDeviceAccelerationStructureFeaturesKHR
             bool accelerationStructure = false;
             bool accelerationStructureCaptureReplay = false;
             bool accelerationStructureIndirectBuild = false;
             bool accelerationStructureHostCommands = false;
             bool descriptorBindingAccelerationStructureUpdateAfterBind = false;
 
-            // RayTracingPipeline
+            //--> VkPhysicalDeviceRayTracingPipelineFeaturesKHR
             bool rayTracingPipeline = false;
             bool rayTracingPipelineShaderGroupHandleCaptureReplay = false;
             bool rayTracingPipelineShaderGroupHandleCaptureReplayMixed = false;
             bool rayTracingPipelineTraceRaysIndirect = false;
             bool rayTraversalPrimitiveCulling = false;
 
-            // Fragment Shader Interlock
+            //--> VkPhysicalDeviceFragmentShaderInterlockFeaturesEXT
             bool fragmentShaderSampleInterlock = false;
             bool fragmentShaderPixelInterlock = false;
             bool fragmentShaderShadingRateInterlock = false;
 
-            // Queries
-            bool allowCommandBufferQueryCopies = false;
-            bool inheritedQueries = false;
 
-            // Buffer Device Address
+            //--> VkPhysicalDeviceBufferDeviceAddressFeaturesKHR
             bool bufferDeviceAddress = false;
+            //VkBool32           bufferDeviceAddressCaptureReplay;
+            //VkBool32           bufferDeviceAddressMultiDevice;
+            
+            //--> Nabla:
+            bool dispatchBase = false;
+            bool drawIndirectCount = false;
+            bool allowCommandBufferQueryCopies = false;
         };
         const SFeatures& getFeatures() const { return m_features; }
 
+        enum E_MEMORY_PROPERTY_FLAGS : uint32_t
+        {
+            EMPF_DEVICE_LOCAL_BIT = 0x00000001,
+            EMPF_HOST_VISIBLE_BIT = 0x00000002,
+            EMPF_HOST_COHERENT_BIT = 0x00000004,
+            EMPF_HOST_CACHED_BIT = 0x00000008,
+            EMPF_LAZILY_ALLOCATED_BIT = 0x00000010,
+            EMPF_PROTECTED_BIT = 0x00000020,
+            EMPF_DEVICE_COHERENT_BIT_AMD = 0x00000040,
+            EMPF_DEVICE_UNCACHED_BIT_AMD = 0x00000080,
+            EMPF_RDMA_CAPABLE_BIT_NV = 0x00000100,
```

Comment:
maybe we should comment these out for now (haven't read the extension spec, unsure what extra validation we should do)